src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/ILGenerator.Mono.cs

```diff
@@ -947,7 +948,9 @@ public virtual void EndExceptionBlock()
             InternalEndClause();
             MarkLabel(ex_handlers[cur_block].end);
             ex_handlers[cur_block].End(code_len);
+#if FALSE
```

Comment:
Isn't the whole point of the way the defines are set up now to make it easy to turn these prints on? This makes it more annoying; is there another option here?

Reply:
Removed